packages/vm/src/instance.rs

```diff
@@ -103,15 +102,15 @@ where
         // Ownership of both key pointer is not transferred to the host.
         env_imports.insert(
             "db_remove",
-            Function::new_native_with_env(store, env.clone(), native_db_remove),
+            Function::new_native_with_env(store, env.clone(), do_remove),
```

Comment:
Here you see the name of the import ("db_remove") and the implementation name `do_remove` do not follow the pattern: `do_<import name>`. It should be `do_db_remove` for consistency. Could you check this for all cases?